acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,702 @@
+    "        def dict_to_str(results_dict) -> str:\n",
+    "            return \"|\".join(f\"{v:.5f}\" for v in results_dict.values())\n",
+    "\n",
+    "        out_title = \"\"\"\n",
+    " -----------------------------------------------------------\n",
+    "| Time profiling stats (in ms):                             |\n",
+    " -----------------------------------------------------------\n",
+    "|   total   |    min    |    max    |    mean    |    std   |\n",
+    "        \"\"\"\n",
+    "        print(out_title)\n",
+    "        for timer in self.timer_list:\n",
+    "            print(f\"{timer.type_str}:\")\n",
+    "            print(dict_to_str(timer.get_results_dict()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Benchmark the pytorch model\n",
+    "\n",
+    "In this part, an `evaluator` with the original pytorch model runs several iterations to benchmark the latency."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "torch_parser = get_parser(meta_config, inference_config)\n",
+    "torch_parser[\"dataset_dir\"] = data_root\n",
+    "torch_parser[\"bundle_root\"] = \"./spleen_ct_segmentation\"\n",
+    "torch_evaluator = torch_parser.get_parsed_content(\"evaluator\")\n",
+    "torch_timer = TimerHandler()\n",
+    "torch_timer.attach(torch_evaluator)\n",
+    "torch_evaluator.run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      " -----------------------------------------------------------\n",
+      "| Time profiling stats (in ms):                             |\n",
+      " -----------------------------------------------------------\n",
+      "|   total   |    min    |    max    |    mean    |    std   |\n",
+      "        \n",
+      "RUN:\n",
+      "15383.70117|15383.70117|15383.70117|15383.70117|0.00000\n",
+      "EPOCH:\n",
+      "15383.00879|15383.00879|15383.00879|15383.00879|0.00000\n",
+      "ITERATION:\n",
+      "13375.14413|220.17316|1528.39758|668.75721|321.72094\n",
+      "FORWARD:\n",
+      "3235.11000|48.62182|433.16605|161.75550|95.97792\n",
+      "PREPARE_BATCH:\n",
+      "1786.98771|0.10448|1091.99280|89.34939|258.17436\n",
+      "POST_PROCESS:\n",
+      "10138.31021|171.44656|1417.09570|506.91551|280.83496\n"
+     ]
+    }
+   ],
+   "source": [
+    "torch_timer.print_results()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Benchmark the TensorRT fp32 and fp16 models\n",
+    "\n",
+    "Update the config parser with Torch-TensorRT models and benchmark the end to end latency. Since the `POST_PROCESS` and `PREPARE_BATCH` part take a lot of time, the acceleration of model computation doesn't have much effect on the end2end latency."
```

Comment:
I would rephrase this sentence:
The `POST_PROCESS` and `PREPARE_BATCH` stages require a considerable amount of time. Although the model forward time is much improved, there is still room for acceleration in reducing the end-to-end latency on this particular MONAI bundle.